### PR TITLE
Update 8.13 release notes with known issue on downsampling

### PR DESCRIPTION
**Pull request by kkrik-es**
Related: #106880

### CODE REVIEW

File: docs/reference/release-notes/8.13.0.asciidoc [hunk 1]
```diff
@@ -3,6 +3,16 @@
 
 Also see <<breaking-changes-8.13,Breaking changes in 8.13>>.
 
+[[known-issues-8.13.0]]
+[float]
+=== Known issues
+
+* Downsampling persistent state fails to load, preventing upgraded nodes from joining the cluster (issue: {es-issue}106880[#106880])
```
Comment (martijnvg) [inline docs/reference/release-notes/8.13.0.asciidoc:10]: `Downsampling persistent state` to `Downsampling persistent tasks`?

**Review by dnhatn — APPROVED** [review]
LGTM. Thanks @kkrik-es.

File: docs/reference/release-notes/8.13.0.asciidoc (hunk 1 above)
Comment (martijnvg) [reply docs/reference/release-notes/8.13.0.asciidoc:10]: Nodes upgraded to 8.13.0 fail to start after attempting to load downsample persistent tasks. Upgraded nodes fail to join the cluster and thus failing the upgrade.

Comment (kkrik-es) [reply docs/reference/release-notes/8.13.0.asciidoc]: Done.

**Pull request merged**